libs/application/templates/children-residence-change/src/forms/ParentBForm.ts

```diff
@@ -15,12 +15,35 @@ export const ParentBForm: Form = buildForm({
   mode: FormModes.APPLYING,
   children: [
     buildSection({
-      id: 'backgroundInformation',
-      title: m.section.backgroundInformation,
+      id: 'approveTermsParentB',
+      title: m.section.effect,
       children: [
-        buildTextField({
-          id: 'placeholderId',
-          title: 'placeholder',
+        buildCustomField({
+          id: 'approveTermsParentB',
+          title: 'Hvaða áhrif hefur breytingin?',
```

Comment:
There is a translation string for this but it seems to be missing here and in the parentA form as well for some reason.
`m.terms.general.pageTitle` 

Reply:
Nice catch, this a copy/paste error 🙈 Fixed now :)